dev-games/box2d/patches/box2d-2.3.1.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/Box2D/CMakeLists.txt b/Box2D/CMakeLists.txt
+index a8ad550..17b3069 100644
+--- a/Box2D/CMakeLists.txt
++++ b/Box2D/CMakeLists.txt
+@@ -26,7 +26,8 @@ if(BOX2D_BUILD_EXAMPLES)
+ 
+   # Testbed and dependencies.
+   find_package(OpenGL REQUIRED)
++if(NOT HAIKU) # Haiku has glut, so no need for freeglut. Disable GLUI because it's not ported.
+   add_subdirectory(freeglut)
+   add_subdirectory(glui)
++endif(NOT HAIKU)
+-  add_subdirectory(Testbed)
+ endif(BOX2D_BUILD_EXAMPLES)
+diff --git a/Box2D/Box2D/CMakeLists.txt b/Box2D/Box2D/CMakeLists.txt
+--- a/Box2D/Box2D/CMakeLists.txt
++++ b/Box2D/Box2D/CMakeLists.txt
+@@ -207,8 +207,8 @@ 
+ 	# install build system hooks for third-party apps
+ 	install(EXPORT Box2D-targets DESTINATION ${LIB_INSTALL_DIR}/Box2D)
+ 
+-        set (BOX2D_INCLUDE_DIR ${CMAKE_INSTALL_PREFIX}/include)
++        set (BOX2D_INCLUDE_DIR /boot/system/develop/headers )
+-        set (BOX2D_INCLUDE_DIRS ${BOX2D_INCLUDE_DIR} )
++        set (BOX2D_INCLUDE_DIRS /boot/system/develop/headers )
+-        set (BOX2D_LIBRARY_DIRS ${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR})
++        set (BOX2D_LIBRARY_DIRS /boot/system/develop/lib )
+         set (BOX2D_LIBRARY Box2D)
+         set (BOX2D_LIBRARIES ${BOX2D_LIBRARY})
+diff --git a/Box2D/Box2D/Box2DConfig.cmake.in b/Box2D/Box2D/Box2DConfig.cmake.in
+--- a/Box2D/Box2D/Box2DConfig.cmake.in
++++ b/Box2D/Box2D/Box2DConfig.cmake.in
+@@ -21,8 +21,8 @@
+ set ( BOX2D_DEFINITIONS  "@BOX2D_DEFINITIONS@" )
+-set ( BOX2D_INCLUDE_DIR  "@BOX2D_INCLUDE_DIR@" )
++set ( BOX2D_INCLUDE_DIR  "/boot/system/develop/headers" )
+-set ( Box2D_INCLUDE_DIRS "@BOX2D_INCLUDE_DIRS@" ) # deprecated
++set ( Box2D_INCLUDE_DIRS "/boot/system/develop/headers" ) # deprecated
+-set ( BOX2D_INCLUDE_DIRS "@BOX2D_INCLUDE_DIRS@" )
++set ( BOX2D_INCLUDE_DIRS "/boot/system/develop/headers" )
+ set ( BOX2D_LIBRARY      "@BOX2D_LIBRARY@" )
+ set ( BOX2D_LIBRARIES    "@BOX2D_LIBRARIES@" )
+-set ( BOX2D_LIBRARY_DIRS "@BOX2D_LIBRARY_DIRS@" )
++set ( BOX2D_LIBRARY_DIRS "/boot/system/develop/lib" )
```

Comment:
And revert all changes to this file, since the previous comment will resolve that.